src/emcpy/plots/__init__.py

```diff
@@ -1 +1,2 @@
 from .scatter import scatter
+from .map2d import map2d
```

Comment:
should we call this file `mapping`.
Importing `map2d` from `map2d` has a funny ring to it.

Reply:
or `maps` instead of `mapping`.

Reply:
I like mapping. From .mapping import map2d. That requires a `git mv map2d.py mapping.py`